src/python/tritonfrontend/_api/_kservegrpc.pyi

```diff
@@ -57,6 +57,7 @@ class KServeGrpc:
         max_connection_age_grace_ms: int
         infer_compression_level: int | Grpc_compression_level
         infer_allocation_pool_size: int
+        max_response_pool_size: int
```

Comment:
Thanks for adding these changes to the type hints! Not relevant to this PR, but just to spread knowledge - I think we can apply a change similar to this PR to remove automate generation of these stub `.pyi` files rather than manually keeping them in sync: https://github.com/triton-inference-server/core/pull/414